x-pack/auditbeat/module/system/host/host.go

```diff
@@ -5,23 +5,144 @@
 package host
 
 import (
+	"bytes"
+	"encoding/binary"
+	"encoding/gob"
+	"io"
 	"net"
+	"strconv"
+	"time"
 
+	"github.com/OneOfOne/xxhash"
+	"github.com/joeshaw/multierror"
 	"github.com/pkg/errors"
 
+	"github.com/elastic/beats/auditbeat/datastore"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/common/cfgwarn"
-	"github.com/elastic/beats/metricbeat/mb"
-
 	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/mb"
 	"github.com/elastic/go-sysinfo"
+	"github.com/elastic/go-sysinfo/types"
 )
 
 const (
 	moduleName    = "system"
 	metricsetName = "host"
```

Comment:
@andrewkroh @tsg Should this be `audit.host`?

Reply:
I'm planning to rename the module in a separate PR.

Reply:
I was planning to namespace the fields only:

```
func init() {
	mb.Registry.MustAddMetricSet(moduleName, metricsetName, New,
		mb.DefaultMetricSet(),
        mb.WithNamespace("system.audit")
	)
}
```

Reply:
With https://github.com/elastic/beats/pull/9393 it will become important that `event.dataset` is unique, meaning it should be `system.audit.process` here. Let's see what tricks we can apply here.

Reply:
+1 on getting the PR in and then as soon as #9393 is also merged, will figure it out.

Reply:
I see how having data from two different metricsets having the same `event.dataset` can be confusing. But I don't think that is different from having the same `event.module` and `event.metricset`.

Still, I could see value in having unique dataset names. Who knows, maybe sometime in the future we might have only datasets, and no more modules/metricsets.

I opened https://github.com/elastic/beats/pull/9499 for namespacing the system module, we can discuss any changes there.